semantic_conventions/metrics/process-runtime-jvm-metrics.yaml

```diff
@@ -87,11 +89,7 @@ groups:
     brief: "Number of executing platform threads."
     instrument: updowncounter
     unit: "{thread}"
-    attributes:
-      - id: daemon
-        brief: "Whether the thread is daemon or not."
-        type: boolean
-        requirement_level: recommended
+    extends: metric_attributes.thread
```

Comment:
Please add the attribute here individually with `ref`. If later we add more attributes to that file, they will all show up here automatically and I think we don't want that.

Reply:
somehow not working - can you help?

```
Index: semantic_conventions/metrics/process-runtime-jvm-metrics.yaml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/semantic_conventions/metrics/process-runtime-jvm-metrics.yaml b/semantic_conventions/metrics/process-runtime-jvm-metrics.yaml
--- a/semantic_conventions/metrics/process-runtime-jvm-metrics.yaml	(revision 8965737826547a304db34497943098e987b3c045)
+++ b/semantic_conventions/metrics/process-runtime-jvm-metrics.yaml	(date 1688120152579)
@@ -89,7 +89,8 @@
     brief: "Number of executing platform threads."
     instrument: updowncounter
     unit: "{thread}"
-    extends: metric_attributes.thread
+    attributes:
+      - ref: thread.daemon
 
   - id: metric.process.runtime.jvm.classes.loaded
     type: metric
```

Reply:
@trask @zeitlinger  and I spoke and we added the `daemon` attribute under the `thread` group inside `[trace/general.yaml`](https://github.com/zeitlinger/semantic-conventions/blob/jvm_metric_attributes/semantic_conventions/trace/general.yaml#L202). All works, but that's still inside the `trace` folder. 

I was thinking if we should carve those attributes out and create a `thread-common.yaml` under `semantic_conventions`, as we did for [http-common](https://github.com/zeitlinger/semantic-conventions/blob/jvm_metric_attributes/semantic_conventions/http-common.yaml)? That way we can share the attributes on traces and metrics.

Reply:
makes sense to me, thx for opening #155